### PR TITLE
Add new rules to ESLint - Whitespaces part 2

### PR DESCRIPTION
**Pull request by wkurniawan07**
Previously (#5297) it involves whitespace around operators and keywords. This time it involves empty lines, trailing whitespaces, and indentation among many others.
- eol-last: all files must end with EOL, it doesn't specify which kind of EOL.
- indent: enforces 4-spaces indentation, it's smart enough to allow parameter alignment for readability.
- no-irregular-whitespace: no non-standard whitespaces like nbsp in the code.
- no-multi-spaces: no multiple whitespaces around operators (e.g `var a = *1;`, where `*` is the extra whitespace)
- no-multiple-empty-lines: self-explanatory
- no-trailing-spaces: self-explanatory, but allows purely empty lines
- spaced-comment: comments must have space after the starting `//` or `/*` and before the ending `*/`.

Not sure if all the rules are worth enforcing though, some may be too restrictive (like the no-multiple-empty-lines). If not enforced at least we fixed the existing problems. @kanghj will need your opinion on this!

### CODE REVIEW

File: src/main/webapp/js/instructorFeedbackEdit.js [hunk 1]
```diff
@@ -1022,19 +1019,19 @@ function addMsqOption(questionNumber) {
...
+              + "</span>"
+              + "<input type=\"text\" name=\"" + FEEDBACK_QUESTION_MSQCHOICE + "-" + curNumberOfChoiceCreated + "\" "
+                      + "id=\"" + FEEDBACK_QUESTION_MSQCHOICE + "-" + curNumberOfChoiceCreated + idSuffix + "\" class=\"form-control msqOptionTextBox\">"
+              + "<span class=\"input-group-btn\">"
+                  + "<button type=\"button\" class=\"btn btn-default removeOptionLink\" id=\"msqRemoveOptionLink\" "
+                          + "onclick=\"removeMsqOption(" + curNumberOfChoiceCreated + "," + questionNumber + ")\" tabindex=\"-1\">"
+                      + "<span class=\"glyphicon glyphicon-remove\"></span>"
+                  + "</button>"
+              + "</span>"
+          + "</div>"
```
Comment (kanghj) [inline src/main/webapp/js/instructorFeedbackEdit.js]: This change actually looks a bit odd?

**Comment by kanghj** [comment]
no-multiple-empty-lines looks unnecessary

File: src/main/webapp/js/instructorFeedbackEdit.js (hunk 1 above)
Comment (wkurniawan07) [inline src/main/webapp/js/instructorFeedbackEdit.js]: The trade-off is you're allowing `var a =  *1` (where `*` is a whitespace), or even `var a = ****1;`. Alternatively we can eslint-disable this section (and many others).

**Comment by wkurniawan07** [comment]
I've de-activated the no-multiple-empty-lines rule. Anything else?

**Pull request merged**